crates/astria-composer/src/config.rs

```diff
@@ -22,6 +22,7 @@ pub struct Config {
     /// The chain ID of the sequencer chain
     pub sequencer_chain_id: String,
 
+    #[allow(clippy::doc_markdown)]
```

Comment:
Replace in favor of putting ticks around it, i.e.:
```suggestion
    /// A list of `<rollup_name>::<url>` pairs
```